varlink/src/server.rs

```diff
@@ -151,6 +151,13 @@ impl Listener {
         }
     }
 
+    pub const fn is_already_accepted(&self) -> bool {
```

Comment:
While I know the rest of the code is not consistent about this, having at least a minimal documentation string I think is a good idea for public APIs.